### DISCUSSION TITLE
Is there a better Select All function?

### DISCUSSION THREAD

**Discussion by developman-akl** [orchidjs/tom-select#62] [Q&A] [answered]
Hi All,

Is there another way to **Select All** options by clicking a button other than going through one by one every options and call `.addItem()` like this?

```
$("#tomSelect > option").each(function() {
    control.addItem(this.value, true);
});
```

The problem with the above approach is it takes forever to populate the input in case there were > 100 options to select.

Is there an inverse of `.clear()` or a similar function I just missed?

Thanks in advance!

**Comment by oyejorge** [comment, marked as answer]
This should work better
```js
let values = Object.keys(control.options);
control.addItems(values);
```

**Reply by developman-akl** [reply]
This is perfect. Cheers!

**Reply by developman-akl** [reply]
PS There is no mention about `.addItems(values)` in the docs, maybe worth including it.
